src/dialogs/more-info/controls/more-info-automation.ts

```diff
@@ -34,7 +34,10 @@ class MoreInfoAutomation extends LitElement {
       </div>
 
       <div class="actions">
-        <mwc-button @click=${this.handleAction}>
+        <mwc-button
+          @click=${this.handleAction}
+          .disabled=${this.stateObj!.state === "unavailable"}
```

Comment:
We should hide it instead

Reply:
Is there an general HA best-practise how to do so?

Reply:
Dont use css

Reply:
`${condition ? true : false }`

Reply:
${state !== unavailable ? html`Button HTML` : "" }

Reply:
Why hide and not disable?

Reply:
I would be fine either way. I assumed performance reasons (less HTML to render)?
@zsarnett Can you chime in?

Reply:
I think disabled tell more to the user than a button missing

Reply:
We disable things when entities are unavailable in general, so we should do it here too.

Reply:
Doesnt the text already say that?

Reply:
Why show a button and text saying its unavailable?

Reply:
So a user knows it normally would be available

Reply:
I agree (e.g. the "enable" switch is also disabled and not hidden) so I changed this PR back to disabling of the "Execute" button.